benchmarking/factual_consistency_ja.ipynb

```diff
@@ -0,0 +1,334 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note book to compute\n",
+    "- The correlation between the `ja.factual_consistency` metric outputs and human annoted consistency scores converting `\"contradiction\" -> 0, \"neutral\" -> 0.5 and \"entailment\" -> 1`\n",
+    "- Ternary classification of the test data by `ja.factual_consistency` with 0.333.., 0.66... as the thresholds\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "JNLI (valid) has 2434 data points\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Load the benchmark datasets\n",
+    "import json\n",
+    "\n",
+    "# These file can be downloaded from JGLUE/JNLI-v1.1\n",
+    "# https://github.com/yahoojapan/JGLUE/tree/main/datasets/jnli-v1.1\n",
+    "# The license is CC-BY-SA-4.0, so I do not include the file to the repository\n",
+    "# itself just in case.\n",
```

Comment:
I think we can remove this? Or you could also just add `(CC-BY-SA-4.0 license)` next to the link the the data if you want to